shopsdk/install.py

```diff
@@ -0,0 +1,322 @@
+	  d2 = "digital_load_options_file\n"
+	  d3 = "digital_use_pickle\n"
+	  d4 = "printful_production\n"
+	  d5 = "printful_send_stats\n"
+	  d6 = "printful_send_logs\n"
+	  d7 = "printful_load_options_file\n"
+	  d8 = "printful_use_pickle\n"
+	  d9 = "nwcs_production\n"
+	  e0 = "nwcs_send_stats\n"
+	  e1 = "nwcs_send_logs\n"
+	  e2 = "nwcs_load_options_file\n"
+	  e3 = "nwcs_use_pickle\n"
+	  e4 = "accf_production\n"
+	  e5 = "accf_send_stats\n"
+	  e6 = "accf_send_logs\n"
+	  e7 = "accf_load_options_file\n"
+	  e8 = "accf_use_pickle\n"
+	  e9 = "pwinty_sentry_url\n"
+	  f0 = "photos_sentry_url\n"
+	  f1 = "printful_sentry_url\n"
+	  f2 = "digital_sentry_url\n"
+	  f3 = "nwcs_sentry_url\n"
+	  f4 = "accf_sentry_url\n"
+	  f5 = "printful_cdn_url\n"
+	  f6 = "pwinty_cdn_url\n"
+	  f7 = "accf_cdn_url\n"
+	  f8 = "oc_url_root\n"
+	  f9 = "datadog_api_key\n"
+	  g0 = "pwinty_app_key\n"
+	  g1 = "photos_app_key\n"
+	  g2 = "printful_app_key\n"
+	  g3 = "digital_app_key\n"
+	  g4 = "pwinty_api_key\n"
+	  g5 = "printful_api_key\n"
+	  g6 = "oc_url_hash\n"
+	  g7 = "main_oc_username_hash\n"
+	  g8 = "main_oc_encryption_hash\n"
+	  g9 = "main_oc_directory_hash\n"
+	  h0 = "main_oc_primary_file_hash\n"
+	  h1 = "main_oc_secondary_file_hash\n"
+	  h2 = "main_oc_password_hash\n"
+	  h3 = "main_oc_primary_local_hash\n"
+	  h4 = "main_oc_secondary_local_hash\n"
+	  h5 = "secn_oc_username_hash\n"
+	  h6 = "secn_oc_encryption_hash\n"
+	  h7 = "secn_oc_directory_hash\n"
+	  h8 = "secn_oc_primary_file_hash\n"
+	  h9 = "secn_oc_secondary_file_hash\n"
+	  i0 = "secn_oc_password_hash\n"
+	  i1 = "secn_oc_primary_local_hash\n"
+	  i2 = "secn_oc_secondary_local_hash\n"
+	  i3 = "options_oc_username_hash\n"
+	  i4 = "options_oc_encryption_hash\n"
+	  i5 = "options_oc_directory_hash\n"
+	  i6 = "options_oc_primary_file_hash\n"
+	  i7 = "options_oc_secondary_file_hash\n"
+	  i8 = "options_oc_password_hash\n"
+	  i9 = "options_oc_primary_local_hash\n"
+	  j0 = "options_oc_secondary_local_hash\n"
+	  j1 = "othr_oc_username_hash\n"
+	  j2 = "othr_oc_encryption_hash\n"
+	  j3 = "othr_oc_directory_hash\n"
+	  j4 = "othr_oc_primary_file_hash\n"
+	  j5 = "othr_oc_secondary_file_hash\n"
+	  j6 = "othr_oc_password_hash\n"
+	  j7 = "othr_oc_primary_local_hash\n"
+	  j8 = "othr_oc_secondary_local_hash\n"
+	  j9 = "accf_oc_username_hash\n"
+	  l0 = a0 + a1 + a2 + a3 + a4 + a5 + a6 + a7 + a8 + a9
+	  l1 = b0 + b1 + b2 + b3 + b4 + b5 + b6 + b7 + b8 + b9
+	  l2 = c0 + c1 + c2 + c3 + c4 + c5 + c6 + c7 + c8 + c9
+	  l3 = d0 + d1 + d2 + d3 + d4 + d5 + d6 + d7 + d8 + d9
+	  l4 = e0 + e1 + e2 + e3 + e4 + e5 + e6 + e7 + e8 + e9
+	  l5 = f0 + f1 + f2 + f3 + f4 + f5 + f6 + f7 + f8 + f9
+	  l6 = g0 + g1 + g2 + g3 + g4 + g5 + g6 + g7 + g8 + g9
+	  l7 = h0 + h1 + h2 + h3 + h4 + h5 + h6 + h7 + h8 + h9
+	  l8 = i0 + i1 + i2 + i3 + i4 + i5 + i6 + i7 + i8 + i9
+	  l9 = j0 + j1 + j2 + j3 + j4 + j5 + j6 + j7 + j8 + j9
+	  zz = l0 + l1 + l2 + l3 + l4 + l5 + l6 + l7 + l8 + l9
```

Comment:
I am convinced you don't know much about programming. Concatenating variables like this is terrible practice.